landing-page/src/index.css

```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+}
+
+a {
+color: #ffffff;
+text-decoration: none;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  text-align: center;
+  color: #ffffff;
+  height: 750px;
+  background-image: url('/src/ph.jpg');
+  background-size:cover;
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-attachment: scroll ;
+}
+
+.header:after {
+  content: "";
+  position: absolute;
+  height: 750px;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.6);
+  z-index: 1;
+}
+
+.headerMid{
+  z-index: 2;
+}
+
+#textTyping{
+  font-weight: 400;
+}
+
+.navBar {
+  width: 100%;
+  color: #ffffff;
+  position: absolute;
```

Comment:
Why do you need an absolute position here?

Reply:
For fixing it over other content of block